Backend/Database/PostgresDbContext.cs

```diff
@@ -27,8 +39,55 @@ public PostgresDbContext(DbContextOptions<PostgresDbContext> options) : base(opt
     /// <param name="modelBuilder"></param>
     protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
-        // Just here as an example as of now
-        // modelBuilder.Entity<Enrollment>()
-        //     .HasIndex(x => x.ActivityId);
+        base.OnModelCreating(modelBuilder);
+
+        // Member → Enrollment (cascade delete)
+        modelBuilder.Entity<Enrollment>()
+            .HasOne(e => e.Member)
+            .WithMany(m => m.Enrollments)
+            .HasForeignKey(e => e.MemberId)
+            .OnDelete(DeleteBehavior.Cascade);
+
+        // Activity → Enrollment (cascade delete)
+        modelBuilder.Entity<Enrollment>()
+            .HasOne(e => e.Activity)
+            .WithMany(a => a.Enrollments)
+            .HasForeignKey(e => e.ActivityId)
+            .OnDelete(DeleteBehavior.Cascade);
```

Comment:
This is double, you can specify foreign keys multiple times if you remove OnDelete (perhaps even with onDelete)